pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -410,11 +410,18 @@ const (
 	// OperationRotateServiceAccountKeyComplete is a constant for an annotation on a Shoot indicating that the
 	// rotation of the service account signing key shall be completed.
 	OperationRotateServiceAccountKeyComplete = "rotate-serviceaccount-key-complete"
+	// OperationRotateETCDEncryptionKey is a constant for an annotation on a Shoot indicating that the
+	// rotation of the ETCD encryption key shall be performed.
+	OperationRotateETCDEncryptionKey = "rotate-etcd-encryption-key"
 	// OperationRotateETCDEncryptionKeyStart is a constant for an annotation on a Shoot indicating that the
 	// rotation of the ETCD encryption key shall be started.
+	// Deprecated: This annotation is deprecated in favour of `rotate-etcd-encryption-key`, which does a full rotation.
```

Comment:
Can we add an _empty line_ between the _docstring_ and the _deprecation_ note, similar to the `pkg/apis/core/types_shoot.go` change ?

```suggestion
	// 
	// Deprecated: This annotation is deprecated in favour of `rotate-etcd-encryption-key`, which does a full rotation.
```